maintenance/checkLastWikiActivity.php

```diff
@@ -42,18 +42,20 @@ public function execute(): void {
 	}
 
 	private function getTimestamp(): int {
+		$default_id = $this->getServiceContainer->getUserFactory->newFromName("MediaWiki default")->getActorId()
```

Comment:
Missing `;` at the end of the line, and missing `()` for a function call to `$this->getServiceContainer()`